docs/html/development/release-process.rst

```diff
@@ -93,8 +93,13 @@ issues by pip's maintainers.
 Python Support Policy
 ---------------------
 
-In general, a given Python version is supported until its usage on PyPI falls below 5%.
-This is at the maintainers' discretion, in case extraordinary circumstances arise.
+pip supports `CPython versions that are not end-of-life`_. Older versions of CPython may
```

Comment:
```suggestion
Pip supports `CPython versions that are not end-of-life`_. Older versions of CPython may
```

Minor nit - I don't think we have a policy that "pip" is always in lower case? Certainly, I don't follow this.

Reply:
I actually followed the pattern of the previous section which started the sentence with a lower case pip.

Reply:
Ah, apparently there is even a written tentative [convention](https://github.com/pypa/pip/blob/main/docs/html/development/conventions.rst#writing) to always write pip lower case.

Reply:
Pretty sure I added that because Paul and I had a disagreement about this. 😂

Reply:
lol yeah, I recall now. I find "all lowercase" conventions to be awkward and (to be blunt) a little pretentious. If "pip" is a proper noun, it should either be capitalised all the time, as `Pip` (like place and person names), or capitalised like any other noun, at the start of a sentence but not otherwise (which is what I prefer). If it's an acronym ("pip installs packages") it should be all-caps ("PIP"), but I don't think that anyone sees it that way.

I'm never going to follow a requirement to lowercase "pip" at the start of a sentence, but I don't care enough to try to force my views on other people, if they prefer to do so.

Reply:
I promise I have no opinion whatsover about this :) I'll merge as is for consistency because I'm lazy.